CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Bugfix: Fixed blocked user list sticking around when switching from a logged in user to being logged out. (#4437)
 - Bugfix: Fixed search popup ignoring setting for message scrollback limit. (#4496)
 - Bugfix: Fixed a memory leak that occurred when loading message history. This was mostly noticeable with unstable internet connections where reconnections were frequent or long-running instances of Chatterino. (#4499)
+- Bugfix: Fixed reply context showing blocked user's name & message (#4502)
```

Comment:
Nitpicking, but this is not a bugfix. There's no expectation set for replies to a blocked user to be hidden or censored. Make this a Minor entry instead, and make sure there's a period at the end of the changelog entry (e.g. `- Minor: asd asd. (#4502)`